index.js

```diff
@@ -34,22 +39,66 @@ function maybeAppendDot( text ){
 
 async function generateTranslations( translatorNames, text, lang, firstChunkOnly ){
 	const extractedText = extract(text);
-	const translations = await Translator.translate(translatorNames, {text: extractedText, to: lang, firstChunkOnly: firstChunkOnly});
+	const translations = {
+		texts : {}, // name -> text
+		translatorNames : [], // names
+		audioUrls : {}, // name -> url
+		audioButtonText : {}, // name -> text
+	};
+
+	translations.texts = await Translator.translate(translatorNames, {text: extractedText, to: lang, firstChunkOnly: firstChunkOnly});
 
-	translations.original = extractedText;
-	translations.outputs = ['original'].concat(translatorNames);
+	translations.texts['original'] = extractedText;
+	translations.translatorNames = ['original'].concat(translatorNames);
 
 	// convert \n\n-separated blocks of text into <p>-wrapped blocks of text
-	translations.outputs.map( translatorName => {
+	translations.translatorNames.map( translatorName => {
 		let textWithParas;
-		if( translations[translatorName].hasOwnProperty('error') ){
-			textWithParas = translations[translatorName];
+		if( translations.texts[translatorName].hasOwnProperty('error') ){
+			textWithParas = translations.texts[translatorName];
 		} else {
-			const textWithBackslashNs = translations[translatorName];
+			const textWithBackslashNs = translations.texts[translatorName];
 			const paras = textWithBackslashNs.split('\n\n').map( para => { return `<p>${para}</p>`});
 			textWithParas = paras.join('\n');
 		}
-		translations[translatorName] = textWithParas;
+		translations.texts[translatorName] = textWithParas;
+	});
+
+	const originalVoice = 'Amy';
+	const originalLang  = 'en';
+	var translationVoice     = originalVoice;
+	var translationVoiceLang = originalLang;
+	var langLC = lang.toLowerCase();
+
+	if (langLC === 'fr') {
+		translationVoice = 'Celine';
+		translationVoiceLang = langLC;
```

Comment:
`langLC` is already `lang.toLowerCase();` is there a reason it's reassigned when `fr` or `de`?